operations/commit_queue.go

```diff
@@ -215,29 +217,57 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 }
 
 func listPRCommitQueueItem(ctx context.Context, item restModel.APICommitQueueItem, projectRef *model.ProjectRef, uiServerHost string) {
+	issue := restModel.FromAPIString(item.Issue)
+	displayPRAuthor(ctx, projectRef, issue)
+
 	prDisplay := `
            PR # : %s
             URL : %s
 `
-	prDisplayVersion := "          Build : %s/version/%s"
-
-	issue := restModel.FromAPIString(item.Issue)
 	url := fmt.Sprintf("https://github.com/%s/%s/pull/%s", projectRef.Owner, projectRef.Repo, issue)
 	grip.Infof(prDisplay, issue, url)
 
+	prDisplayVersion := "          Build : %s/version/%s"
 	if restModel.FromAPIString(item.Version) != "" {
 		grip.Infof(prDisplayVersion, uiServerHost, restModel.FromAPIString(item.Version))
 	}
+
 	grip.Info("\n")
 }
 
+// if we can display author, we do, but in the event of an error we skip it
+func displayPRAuthor(ctx context.Context, projectRef *model.ProjectRef, issue string) {
+	conf, err := evergreen.GetConfig()
```

Comment:
I don't think you can call this from the CLI. I think you have to pass the author in the API call to the server.

Reply:
Hmm I might be understanding your comment wrong, but the person calling `evergreen commit-queue list` isn't the author, it's the person who created the PR, so we'll have to get the PR somehow. I could write a Communicator for the existing Connector `GetGithubPR`? But is that an API route that users can have access to then because I don't think they should be allowed to.

Reply:
maybe the API could return just the author for a PR? (and could then lookup their Evergreen username)

Reply:
> maybe the API could return just the author for a PR? (and could then lookup their Evergreen username)

Does it even need to look up their Evergreen username?

Reply:
> Hmm I might be understanding your comment wrong, but the person calling evergreen commit-queue list isn't the author, it's the person who created the PR, so we'll have to get the PR somehow. I could write a Communicator for the existing Connector GetGithubPR? But is that an API route that users can have access to then because I don't think they should be allowed to.

I just mean that `evergreen.GetConfig()` isn't possible from the CLI, only on the app server.

Reply:
Not necessarily, but it might be nice.